apps/builder/app/canvas/collaborative-instance-react.ts

```diff
@@ -0,0 +1,6 @@
+import { useEffect } from "react";
+import { updateCollaborativeInstanceRect } from "./collaborative-instance";
+
+export const useUpdateCollaborativeInstanceRect = () => {
+  useEffect(updateCollaborativeInstanceRect, []);
```

Comment:
Just put it into canvas, too much repetition.

Reply:
?

Reply:
I mean see where you use the hook. Just add `useEffect(hook, [])` on canvas